core/schema/sdk.go

```diff
@@ -91,6 +91,8 @@ func (s *moduleSchema) builtinSDK(ctx context.Context, root *core.Query, sdkName
 		return s.loadBuiltinSDK(ctx, root, sdkName, digest.Digest(os.Getenv(distconsts.PythonSDKManifestDigestEnvName)))
 	case "typescript":
 		return s.loadBuiltinSDK(ctx, root, sdkName, digest.Digest(os.Getenv(distconsts.TypescriptSDKManifestDigestEnvName)))
+	case "elixir":
+		return s.loadBuiltinSDK(ctx, root, sdkName, digest.Digest(os.Getenv(distconsts.ElixirSDKManifestDigestEnvName)))
```

Comment:
> let's not bundle community maintained SDKs in the engine container just yet. We may revisit at a later date, but for now users should use the git path to the runtime container in `--sdk`.

Agreed. Testing the runtime is quite slow because I need to do build the engine & cli while external runtime experience is much better. So I'll revert the change. 👍 



> There's been some ideas floating around for improving this, for example, by reserving a shorthand prefix (instead of requiring the full git ref) in order to be easier to type and fetch the sources at a standard location by URL convention, but the design for this isn't yet flushed out.

I have been thinking about having a sdk & runtime under dagger organization rather than having under dagger repository. This would make sdk maintainer iterate more faster since they have more control on that repository. But the release process would be more complex.

Reply:
Do we need to remove elixir test out of integration test or need to switch to git path?

Reply:
> I have been thinking about having a sdk & runtime under dagger organization rather than having under dagger repository. This would make sdk maintainer iterate more faster since they have more control on that repository. But the release process would be more complex.

Yeah, this has been suggested before (cc @gerhard) for that exact reason (also peformance would be better for the runtime module when loading for the first time), and same downside too.

There's also a balance in terms of control over the design. On one hand it allows faster iteration due to control, but on the other hand, having the team review changes helps in keeping alignment. The main problems are low bandwidth and probably lack of familiarity with the language.

Reply:
> Do we need to remove elixir test out of integration test or need to switch to git path? My opinion is temporary moving it out for now would be better. Adding to integration tests is quite a bit painful, especially finding log in the CI. 😂
> 
> However, I'm thinking about having a conformance test, like https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md. I see that in integration tests already have this kind of test but didn't expose the spec outside.

Yeah, I expect we'll want to have an easier system for integration tests for multiple SDKs, where you just write the code for modules that have to conform to certain expectations.

We'll feel the pressure for that more and more, as we need to test all the SDKs and more may be coming. \cc @jedevc

As for moving the tests out, yes if it's easier for you. Python and TypeScript have their own test suite in their own `sdk/<name>` dirs. And both have a `dev` module, written in their language, to help manage the SDK.

The idea is that you can call the dev module manually, but also add it as a dependency to the root's ci module, which then simply calls the dev module's functions rather than all the code being there.

It also helps with observability and concurrency when done like this.

Reply:
> As for moving the tests out, yes if it's easier for you. Python and TypeScript have their own test suite in their own `sdk/<name>` dirs. And both have a `dev` module, written in their language, to help manage the SDK.

I believe `dev` module is a good dogfood for the SDK itself. I'll go in that way. 👍

Reply:
@helderco I think it's done now. Let me know if you have something to change. 🙇